projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -17,30 +17,105 @@
 
 <ng-container *ngIf="record">
   <header class="row">
-    <h1>{{ record.metadata.title }}</h1>
+
+    <!-- TITLE -->
+    <ng-container *ngIf="record.metadata.title">
+      <ng-container *ngFor="let title of record.metadata.title">
+        <ng-container *ngIf="title.type === 'bf:Title'">
+          <h1>{{ title._text }}</h1>
+        </ng-container>
+      </ng-container>
+    </ng-container>
   </header>
   <section>
     <dl class="row mb-0">
+
+      <!-- ALTERNATIVE GRAPHICS -->
+      <ng-container *ngIf="record.metadata.ui_title_altgr">
+        <ng-container *ngFor="let altgr_title of record.metadata.ui_title_altgr; first as isFirst">
+          <ng-container *ngIf="isFirst; else elseBlock">
+            <dt class="col-sm-4 offset-sm-2 offset-md-0 label-title" translate>
+              Title
+            </dt>
+            <dd class="col-sm-6 col-md-8 mb-0">
+              {{ altgr_title | titlecase }}
+            </dd>
+          </ng-container>
+          <ng-template #elseBlock>
+            <dt class="col-sm-4 offset-sm-2 offset-md-0">
+              &nbsp;
+            </dt>
+            <dd class="col-sm-6 col-md-8 mb-0">
+              {{ altgr_title | titlecase }}
+            </dd>
+          </ng-template>
```

Comment:
I think this could be simplified with something like this:

```
<dt class="col-sm-4 offset-sm-2 offset-md-0" translate>
    <span class="label-title" *ngIf="isFirst; else elseBlock">Title</span>
    <ng-template #elseBlock>&nbsp;</ng-template>
</dt>
<dd class="col-sm-6 col-md-8 mb-0">
  {{ altgr_title | titlecase }}
</dd>
```

But I don't know what label-title class is really doing

Reply:
@sebastiendeleze `label-title` is used to add a semicolon (:) after the tag content. It is useful to use the `translate` into the tag instead of use the `translate pipe`

Using your proposal, the "extract_messages" command will populate translation file with all the 'dt' content (span tag included). The translate tag will be place into the 'span' opening tag.